content/blog/2019-11-13-space-efficient-gc/index.md

```diff
@@ -0,0 +1,100 @@
++++
+title = "Space Efficient Conservative Garbage Collection"
+extra.author = "Yi Jiang"
+extra.bio = """
+  [Yi Jiang](http://www.cs.cornell.edu/~yijiang/) is a 2nd year Ph.D. student interested in computer architecture and systems.
+"""
+latex = true
+
++++
+
+["Space efficient conservative garbage collection"](https://dl.acm.org/citation.cfm?id=155109) describes some inexpensive but useful techniques that can make *conservative garbage collectors* more space efficient.
+Their techniques can reduce pointer misidentification to retain less memory during runtime, and also excess retention due to less information about variable liveness. Their methods can be easily incorporated into any garbage collecting allocator transparently to client programs.
```

Comment:
"and also excess retention due to less information about variable liveness": I think there's a verb missing here? Perhaps "and can *prevent* excess retention"? I'm also not clear here about _who_ has "less information about variable liveness." Is the idea that the _baseline_ doesn't have _enough_ information?